packages/node-integration-tests/suites/express/sentry-trace/baggage-header-assign/test.ts

```diff
@@ -14,39 +14,39 @@ test('Should assign `baggage` header which contains 3rd party trace baggage data
   expect(response).toMatchObject({
     test_data: {
       host: 'somewhere.not.sentry',
-      baggage: expect.stringContaining('foo=bar,bar=baz'),
+      baggage: 'foo=bar,bar=baz,sentry-environment=prod,sentry-release=1.0',
     },
   });
 });
 
-test('Should assign `baggage` header which contains sentry trace baggage data of an outgoing request.', async () => {
+test('Should assign `baggage` header which contains sentry trace baggage data to an outgoing request.', async () => {
```

Comment:
Can we say in this test title explaining, that we don't overwrite baggage that's already on the outgoing request?

Reply:
Yes, thanks for bringing this up. Changed this title and the one below it
(this makes me think that I should add more tests covering different mutability cases here in #5205)

Reply:
dd44ae74e9a3d9ddf5ef0f3a4087a93c7bd179ef